src/styles/blocks/card.scss

```
@@ -0,0 +1,104 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__pic {
+    height: 134px;
+    width: 160px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin: 0 0 4px;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    width: 100%;
+    font-weight: 400;
+    color: #616070;
+    font-size: 10px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    margin-bottom: 24px;
+    width: 100%;
+    font-weight: 400;
+    color: #060b35;
+    font-size: 10px;
+  }
+
+  &__price {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    margin-bottom: 16px;
+    justify-content: space-between;
+    width: 100%;
+    font-weight: 400;
+    color: #616070;
+    font-size: 12px;
+  }
+
+  &__price-number {
+    font-weight: 700;
+    color: #060b35;
+    font-size: 16px;
+  }
+
+  &__buy {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 12px;
+    width: 100%;
+    border-radius: 5px;
+    margin-top: 10px;
+    background-color: #00acdc;
+    color: white;
+    border: 1px solid #00acdc;
+    text-decoration: none;
+    line-height: 16px;
+    font-size: 14px;
+    text-transform: uppercase;
+  }
+
+  &__buy:hover {
+    background-color: white;
+    color: #00acdc;
+    border: 1px solid #00acdc;
+  }
+}
+
+.stars {
+  display: flex;
+  top: 200px;
+
+  &__star {
+    background-image: url(./images/star-active.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    height: 16px;
+    width: 16px;
```

```suggestion
    width: 16px;
    margin-right: 4px;
```
there should be a margin between stars according to mockup